packages/roosterjs-editor-plugins/lib/ContentEdit/ContentEditFeatures.ts

```diff
@@ -131,5 +139,9 @@ export function getDefaultContentEditFeatures(): ContentEditFeatures {
         noCycleCursorMove: Browser.isChrome,
         smartOrderedList: false,
         smartOrderedListStyles: ['lower-alpha', 'lower-roman', 'decimal'],
+        bold: true,

```

Comment:
we probably would not include these in the default enabled set, and instead pass them in to explicitly enable them